CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Attention: The newest changes should be on top -->
 
 ### Added
 
+- TST: Add branch coverage for fin_flutter_analysis and fix keyword argument bug [#904](https://github.com/RocketPy-Team/RocketPy/pull/815)
```

Comment:
We don't add TST pull requests to the CHANGELOG.md

```suggestion
```

Reply:
<img width="706" height="583" alt="image" src="https://github.com/user-attachments/assets/25d9a992-b68d-4239-a84b-54410f1a262c" />